dbusmock/templates/ofono.py

```diff
@@ -47,6 +49,19 @@ def load(mock, parameters):
         mock.AddModem(parameters.get('ModemName', 'ril_0'), {})
 
 
+def newSimSerial():
+    global _simSerialCounter
+    serial = 'ed752c5f-f723-437e-bc6c-' + ('%012d' % _simSerialCounter)
```

Comment:
This this serial magic in any way? I. e. does ofono do something special for this prefix? If so, please add a comment. If it's just a random number, maybe 12345..abcd? (but that's just a nitpick)